From e724e1d8a4998b64a99a288f9cbfba368eeea38e Mon Sep 17 00:00:00 2001
Message-Id: <e724e1d8a4998b64a99a288f9cbfba368eeea38e.1691350701.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1691350701.git.sharlatanus@gmail.com>
References: <cover.1691350701.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Sun, 6 Aug 2023 19:45:24 +0100
Subject: [PATCH 7/9] gnu: Add go-github-com-nsqio-go-diskqueue.

* gnu/packages/golang.scm (go-github-com-nsqio-go-diskqueue): New variable.
---
 gnu/packages/golang.scm | 22 ++++++++++++++++++++++
 1 file changed, 22 insertions(+)

diff --git a/gnu/packages/golang.scm b/gnu/packages/golang.scm
index fd3807bb7e..637e6ebe73 100644
--- a/gnu/packages/golang.scm
+++ b/gnu/packages/golang.scm
@@ -3183,6 +3183,28 @@ (define-public go-github-com-nathan-osman-go-sunrise
 sunset times from geographical coordinates and a date.")
       (license license:expat))))
 
+(define-public go-github-com-nsqio-go-diskqueue
+  (package
+    (name "go-github-com-nsqio-go-diskqueue")
+    (version "1.1.0")
+    (source (origin
+              (method git-fetch)
+              (uri (git-reference
+                    (url "https://github.com/nsqio/go-diskqueue")
+                    (commit (string-append "v" version))))
+              (file-name (git-file-name name version))
+              (sha256
+               (base32
+                "1hp66hkmfn0nyf3c53a40f94ah11a9rj01r5zp3jph9p54j8rany"))))
+    (build-system go-build-system)
+    (arguments
+     '(#:import-path "github.com/nsqio/go-diskqueue"))
+    (home-page "https://github.com/nsqio/go-diskqueue")
+    (synopsis "Go package providing a filesystem-backed FIFO queue")
+    (description
+     "This package provides a Go package providing a filesystem-backed FIFO queue")
+    (license license:expat)))
+
 (define-public go-github-com-hebcal-gematriya
   (let ((commit "fe3043f73e415eb82727701d10f2fb40f87675e9")
         (revision "0"))
-- 
2.40.1

